surefire-providers/surefire-junit-platform/src/main/java/org/apache/maven/surefire/junitplatform/RunListenerAdapter.java

```diff
@@ -243,7 +243,7 @@ private StackTraceWriter getStackTraceWriter( TestIdentifier testIdentifier, Thr
     {
         String className = getClassName( testIdentifier );
         String methodName = getMethodName( testIdentifier ).orElse( "" );
-        return new PojoStackTraceWriter( className, methodName, throwable );
+        return new LegacyPojoStackTraceWriter( className, methodName, throwable );
```

Comment:
Do you have time for this? We talked about it on Slack.

Reply:
What was the solution?
- restore `PojoStackTraceWriter` and use `common-java5` artifact or
- copy `PojoStackTraceWriter` to a local `StackTraceWriter`?

Reply:
Yes to restore please. We should use the previous `PojoStackTraceWriter`.
The old `LegacyPojoStackTraceWriter` has bad user experience on the console. Now it has only one purpose within plugin's process. Maybe it could be moved to `surefire-common`. We will have a chance to break backwards compatibility in 3.0.0.

Reply:
Done.